docs/life_of_a_dataset.md

```diff
@@ -74,6 +78,10 @@ In the PR, please also include the validation results (`report.json` and `summar
 
 If you introduced new statistical variables, please create a Pull Request for them in the [schema repo](https://github.com/datacommonsorg/schema).
 
+## Manifesting Artifacts
+
+After your PR is reviewed and approved, the Data Commons team will work with you to manifest the artifacts you've created in our internal system.
```

Comment:
nit: should we also state that only after the artifacts are manifested will they be added to the KG and be accessible?